preview/test-files/ver.payload.json

```diff
@@ -0,0 +1,21 @@
+{  
+   "data":{  
+      "attributes":{  
+         "custom":{  
+            "repo_url":"git@github.com:testrepo/testproject.git",
+            "version_updates":[  
+               {  
+                  "version":"3.4.1",
+                  "latest_version":"3.4.2",
+                  "name":"io.vertx:vertx-web",
+                  "ecosystem":"maven"
```

Comment:
`ecosystem` this is not used in template file (body.html).
Is this valid attribute?
If yes, use it in body.html
If no, remove it from here.